covalent/executor/base.py

```diff
@@ -227,7 +231,7 @@ def execute_in_conda_env(
         shell_commands += f'os.remove("{temp_filename}")\n\n'
 
         shell_commands += "fn = function.get_deserialized()\n"
-        shell_commands += f"result = fn(**{kwargs})\n\n"
+        shell_commands += f"result = fn(*{args}, **{kwargs})\n\n"
```

Comment:
should this instead be `fn({*args}, {**kwargs})`?  or is this the correct syntax?

Reply:
So since this is an fstring it will just add the list and dictionary which will then get unwrapped when the `shell_commands` get executed. `{}` here are functionally irrelevant.